charts/metal3/0.2.0/charts/ironic/values.yaml

```diff
@@ -16,28 +15,29 @@ global:
 
   # whether to enable PXE boot capability
   # NOTE: enable_dnsmasq must set to 'true' in order for this option to be effective.
-  enable_pxe_boot: true
+  enable_pxe_boot: false
 
   # DNS Policy of the Ironic pod
   dnsPolicy: "ClusterFirstWithHostNet"
 
   # IP address of the router associated with the specified DHCP
   # address range
-  dnsmasqDefaultRouter: 192.168.21.254
-  
+  dnsmasqDefaultRouter: ""
+
   # IP address of the dns server to be provided with DHCP
   # response
-  dnsmasqDNSServer: 192.168.20.5
+  dnsmasqDNSServer: ""
```

Comment:
It would be a good idea to also add validation whether these properties are empty inside of Ironic's [ConfigMap](https://github.com/suse-edge/charts/blob/main/charts/metal3/0.2.0/charts/ironic/templates/configmap.yaml).

That way we will not ship ConfigMaps with empty properties inside them. Furthermore if clusters have validation web hooks for empty properties in ConfigMpas this will break our chart deployment

The properties that would need to be checked in Ironic's [ConfigMap](https://github.com/suse-edge/charts/blob/main/charts/metal3/0.2.0/charts/ironic/templates/configmap.yaml) would be: `DNSMASQ_DEFAULT_ROUTER`, `DNSMASQ_DNS_SERVER_ADDRESS` , `dhcpRange` , `PROVISIONING_INTERFACE` and `PROVISIONING_IP`.